src/proto/grpc/testing/messages.proto

```diff
@@ -202,8 +202,13 @@ message LoadBalancerStatsRequest {
 }
 
 message LoadBalancerStatsResponse {
+  message RPCsByPeer {
```

Comment:
This type is changed to `RpcsByPeer`. Please update to keep in sync with https://github.com/grpc/grpc/pull/23436/files

Reply:
OK will update

Reply:
and will update to rpcs_by_method